test/cbmc/proofs/Makefile-project-defines

```diff
@@ -27,9 +27,10 @@ COMPILE_FLAGS += -fPIC -std=gnu90
 #
 INCLUDES += -I$(CBMC_ROOT)/include
 INCLUDES += -I$(SRCDIR)/source/include
+INCLUDES += -I$(SRCDIR)/test/include
 
 # Preprocessor definitions -D...
-DEFINES += -DFLEET_PROVISIONING_DO_NOT_USE_CUSTOM_CONFIG
+# DEFINES =
```

Comment:
Why is this commented out?

Reply:
It is removed as we are providing a config and not defining FLEET_PROVISIONING_DO_NOT_USE_CUSTOM_CONFIG

Reply:
CBMC does not necessarily need the library config to be provided just for logging, does it?

Reply:
No, but without it, the code within the macro calls will not be checked by cbmc

Reply:
We usually do not include logging calls within the purview of CBMC analysis, but I'm fine with including those as well.